src/CalculatorUITests/StandardModeFunctionalTests.cs

```diff
@@ -304,5 +314,893 @@ public void Memory_ClearTest()
+        }
+        [TestMethod]
+        public void Error_CubedNegativeNumber()
+        {
+            page.StandardOperators.NumberPad.Input(9);
+            page.StandardOperators.NegateButton.Click();
+            page.StandardOperators.SquareRootButton.Click();
+            Assert.IsTrue(page.GetCalculatorResultText() == "Invalid input");
+        }
+        #endregion
+
+        #region KOT(Keep on top) Tests
+        [TestMethod]
+        [Priority(0)]
+        public void KOT_EnterExitKeepOnTop()
+        {
+            /// <summary>
+            /// Test Case 23456961: BVT: KOT: Verify entering and exiting KOT mode
+            /// This automated test verifies the caculator can enter KOT mode and that the window within KOT size range
+            /// and it verifies exiting KOT mode
+            /// </summary>
+            page.StandardKoTMode();
+            page.KOTWindowSizeWithinRange();
+            page.StandardNonKoTMode();
+
+        }
+
+        [TestMethod]
+        [Priority(0)]
+        public void KOT_Tootip()
+        {
+            /// <summary>
+            /// Test Case 23456329: BVT: KOT: Verify tooltip
+            /// This automated test verifies english KOT button tooltips
+            /// </summary>
+            Assert.AreEqual("Keep on top", page.GetKOTToolTipText());
+            page.StandardKoTMode();
+            Assert.AreEqual("Back to full view", page.GetKOTToolTipText());
+        }
+
+        [TestMethod]
+        [Priority(1)]
+        public void KOT_NoMemoryFunction()
+        {
+            /// <summary>
+            /// Test Case 23458498: BVT: KOT: Verify Memory function is not accessible while in KOT mode
+            /// This automated test verifies memory function cannot be triggered in KOT calculator
+            /// </summary>
+            ///
+            page.StandardOperators.NumberPad.Num9Button.Click();
+            page.StandardOperators.MinusButton.Click();
+            page.StandardOperators.NumberPad.Num3Button.Click();
+            page.StandardKoTMode();
+            page.Window.SendKeys(Keys.Enter);
+            page.Window.SendKeys(Keys.Control + "P" + Keys.Control);
+            page.StandardNonKoTMode();
+            page.MemoryPanel.OpenMemoryPanel();
+            Assert.IsNotNull(WinAppDriver.Instance.CalculatorSession.FindElementByAccessibilityId("MemoryPaneEmpty"));
+        }
+
+        [TestMethod]
+        [Priority(1)]
+        public void KOT_HistoryFunction()
+        {
+            /// <summary>
+            /// Test Case 23458436: BVT: KOT: Verify History is not accessible while in KOT mode, but it is still tracked
+            /// This automated test verifies the history flyout cannot be opened in KOT mode and
+            /// that history is tracked while in KOT mode and avalible upon ext of KOT mode
+            /// </summary>
+            page.HistoryPanel.ClearHistory();
+            page.StandardOperators.NumberPad.Num3Button.Click();
```

Comment:
I don't think you need to do these to verify you can't open the history panel. Also the testcase name should be updated to somthing like AOT_HistoryPanelCannotBeOpenedInAOTMode